projects/go-style-guide/src/app/features/ui-kit/components/configuration-docs/configuration-docs.component.html

```diff
@@ -54,81 +84,33 @@ <h3 class="go-heading-3">Update Branding Color</h3>
 
   <go-card class="go-column go-column--100" id="header-branding">
     <ng-container go-card-header>
-      <h2 class="go-heading-2">Enable Header Branding</h2>
+      <h2 class="go-heading-2">Enable Client Branding</h2>
     </ng-container>
     <div class="go-container" go-card-content>
 
-      <div class="go-column go-column--100">
-        <p class="go-body-copy go-body-copy--no-margin">
-          The <code class="code-block--inline">public toggleHeaderBrandingEnabled(): void</code> method can 
-          be used to enable/disable header branding.
+      <div class="go-column go-column--50">
+        <p class="go-body-copy">
+          To enable the "Client Branding" mode, the config on the service must be reset.
```

Comment:
Should we provide a method in the service to just set the `brandingMode`? We provide a method to set the `logo` and the `brandColor`, so for consistency sake should we also provide a way to change the branding mode with out resetting the entire config?

Reply:
Sure, I think the use case for this will be much less because if you're changing the mode you're probably also changing the logos and brand color.. but it can't hurt to throw a couple lines of code in there just in case.